bot/cogs/moderation/utils.py

```diff
@@ -142,25 +154,23 @@ async def notify_infraction(
     """DM a user about their new infraction and return True if the DM is successful."""
     log.trace(f"Sending {user} a DM about their {infr_type} infraction.")
 
-    text = textwrap.dedent(f"""
-        **Type:** {infr_type.capitalize()}
-        **Expires:** {expires_at or "N/A"}
-        **Reason:** {reason or "No reason provided."}
-    """)
+    text = INFRACTION_DESCRIPTION_TEMPLATE.format(
+        type=infr_type.capitalize(),
+        expires=expires_at or "N/A",
+        reason=textwrap.shorten(reason, 1000, placeholder="...") if reason else "No reason provided."
+    )
```

Comment:
@lemonsaurus @ks129 My feedback on this PR has been addressed but a somewhat out-of-scope change was made here that I'm not entirely excited about & @sco1 agreed that it's sub-optimal when we spoke about it some time back.

I requested only that the newlines are removed from the string because they are later on stripped by the `shorten`, and so the result string looks different than what the literal implies, which is misleading. @ks129 made a more adventurous attempt at keeping the newlines by only running the `reason` kwarg through the `shorten`, which helps in the sense that the newlines now remain, but it's only because we're no longer shortening the full string, but only one part of it, and taking an educated guess at best when choosing the length (`1000`) with the justification being that the other fields should never be long enough for this to exceed the maximum length. I'm finding this approach somewhat fragile and I think that a module that our entire moderation suite relies on deserves a more sophisticated approach.

Reply:
I solved this issue by force-shortening (ignoring word breaks) in case when description will be still too long.

Reply:
I think this approach is much better, but why are we still running the `reason` through `textwrap.shorten`? That shouldn't be necessary anymore, right?

Reply:
I think this should be still there, because this try first soft-shortening (breaking between words) and when after this string is still too long, this force-break, what may break word.

Reply:
That is true, but it still suffers from the problem of guessing at what the maximum length should be. I think that for the sake of simplicity and transparency, just the final cut-off would be fine (also easier to make assertions about in the tests). I don't personally see much appeal in keeping whole words if a sentence gets cut off anyway.

Reply:
This is now done @kwzrd

Reply:
Thanks, looks excellent. I will test tomorrow before I merge but I think this is now ready.